packages/api/src/derives/staking/stakingAccount.ts

```diff
@@ -0,0 +1,100 @@
+import {ApiInterfaceRx} from '@cennznet/api/types';
+import {DerivedStakingQuery} from '@polkadot/api-derive/types';
+import {memo} from '@polkadot/api-derive/util';
+import {createType, Option, TypeRegistry, Vec} from '@polkadot/types';
+import {
+  AccountId,
+  Exposure,
+  Keys,
+  Nominations,
+  RewardDestination,
+  StakingLedger,
+  ValidatorPrefs,
+} from '@polkadot/types/interfaces';
+import {ITuple} from '@polkadot/types/types';
+import {combineLatest, Observable, of} from 'rxjs';
+import {map, switchMap} from 'rxjs/operators';
+
+type MultiResultV2 = [
+  Option<AccountId>,
+  Option<Nominations>,
+  RewardDestination,
+  Exposure,
+  [ValidatorPrefs],
+  Option<Keys>,
+  Option<StakingLedger>
+];
+
+function unwrapSessionIds(
+  stashId: AccountId,
+  queuedKeys: [AccountId, Keys][],
+  nextKeys: Option<Keys>
+): {nextSessionIds: AccountId[]; sessionIds: AccountId[]} {
+  const sessionIds = (queuedKeys.find(([currentId]): boolean => currentId.eq(stashId)) || [
```

Comment:
Perhaps it could be more readable to write this piece as:
`const sessionIds: AccountId[] = [];
const idKeys = queuedKeys.find(([id, keys]): boolean => id.eq(stashId));
if (idKeys) {
   sessionIds = idKeys[1];
}`

Reply:
done